repose-aggregator/src/docs/asciidoc/architecture/system-model.adoc

```diff
@@ -32,6 +32,7 @@ Filter::
 An HTTP message interceptor that provides a specific piece of business functionality.
 A filter contains a *Repose* specific named reference.
 
+// <<filter-chain.adoc#,Filters (Filter Chain)>>::
```

Comment:
Huh? Why is this commented out?

Reply:
OIC. You don't want the filter chain doc to be reachable (at least by navigation) until it goes into effect?

Reply:
Correct